examples/installation/.env

```diff
@@ -1 +1 @@
-NUSSKNACKER_VERSION=1.15.1
+NUSSKNACKER_VERSION=1.15.3
```

Comment:
1. How does this bump relate to the synchronization of external repo https://github.com/TouK/nussknacker-installation-example? I thought this had to be merged first into release branch and then released through GH action? But now I see that in that repo it has version 1.15.3.

2. When should this be bumped? Is it part of the release procedure?

Reply:
The pipeline script, during publishing, should set the proper version. The e2e tests also set the current build version. TBH we don't need it. I'm removing it.